authenticity-collector/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
please use this prettier config so it is the same as in our other projects:

```
{
	"semi": true,
	"trailingComma": "none",
	"singleQuote": true,
	"printWidth": 140,
	"tabWidth": 2,
	"useTabs": true
}

```
then please run `npm run format`